miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -703,8 +726,9 @@ private void writeLibraryConfiguration(ObjectMapper mapper, ObjectNode config) {
 
   @RequestMapping(value = "/bulk/propagate", method = RequestMethod.GET)
   public ModelAndView propagateFromSamples(@RequestParam("ids") String sampleIds, @RequestParam("replicates") int replicates,
-      ModelMap model) throws IOException, MisoNamingException {
-    return libraryBulkPropagateBackend.propagate(sampleIds, replicates, model);
+      @RequestParam(name = "sort", required = false) String sort, ModelMap model) throws IOException, MisoNamingException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.core.exception.MisoNamingException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)